internal/formatter/roles_full.go

```diff
@@ -36,7 +36,7 @@ const (
 	userStateHeader           = "State"
 	defaultRoleApiKeysListing = "table {{.ApiKeyName}}\t{{.ApiKeyIssuer}}\t{{.ApiKeyStatus}}"
 	apiKeyNameHeader          = "Name"
-	apiKeyIssuerHeader        = "Issuer"
+	apiKeyIssuerHeader        = "Created By"
```

Comment:
Is this intended?

Reply:
Yeah, to match the labels on API keys tab in UI
And also to have the same nomenclature while listing associated API keys when describing a role